packages/flutter/lib/src/material/chip.dart

```diff
@@ -423,7 +430,8 @@ abstract class DisabledChipAttributes {
   /// Defaults to true. Cannot be null.
   bool get isEnabled;
 
-  /// Color to be used for the chip's background indicating that it is disabled.
+  /// Color used for the chip's background to indicate that it is not
```

Comment:
nit: 
```suggestion
  /// The color used for the chip's background to indicate that it is not
```
(Doesn't seem like a complete sentence otherwise)

Reply:
Done.